e2e/atlas_clusters_test.go

```diff
@@ -49,27 +49,35 @@ func TestAtlasClusters(t *testing.T) {
 		cmd := exec.Command(cliPath,
 			atlasEntity,
 			clustersEntity,
-			"create", clusterName,
+			"create",
+			clusterName,
 			"--region=US_EAST_1",
 			"--members=3",
-			"--instanceSize=M5",
+			"--instanceSize=M10",
```

Comment:
we can't update M2 or M5

Reply:
[q] Were these tests passing before? - Wondering why we didn't catch this until now.

Reply:
yes because we weren't trying to update the cluster, that's new, when I try to update the cluster I created it would fail if is anything bellow an m10